packages/pointer_interceptor/pointer_interceptor_web/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## 0.9.0
```

Comment:
0.0.1.

Reply:
stuart asked me to use 0.9.0 since the main package is also 0.9.0 (tho i guess it'll be 0.10.0 now) - is there a versioning guide or something for this case?
https://github.com/flutter/packages/pull/5233#discussion_r1388478235

Reply:
ah ok! Good to know